app/controllers/api/v1/reservation_controller.rb

```diff
@@ -1,38 +1,45 @@
 class Api::V1::ReservationController < ApplicationController
   before_action :authorize
   def index
-    @reservations = Motorcycle.left_outer_joins(:reservations)
-      .select('reservations.*, motorcycles.*, reservations.id as reservation_id')
+    @my_reserves = Motorcycle.left_outer_joins(:reservations)
```

Comment:
- [ ] It would be nice if you could move some controller logic to your models.  Look at this [example](https://rails-bestpractices.com/posts/2010/07/21/move-model-logic-into-the-model/).

That way, your controllers will be slimmer (this is a best practice). 💪